packages/axios-client-adapter/src/httpClient.ts

```diff
@@ -44,18 +46,22 @@ export class HttpClient {
       timeout = DEFAULT_TIMEOUT,
       httpAgent,
       httpsAgent,
+      proxySettings,
     }: {
       clientConfigOverrides?: AxiosRequestConfig;
       timeout?: number;
       httpAgent?: any;
       httpsAgent?: any;
+      proxySettings?: ProxySettings;
     } = {}
   ) {
+    this._proxySettings = proxySettings;
     this._timeout = timeout;
     this._axiosInstance = axios.create({
       ...DEFAULT_AXIOS_CONFIG_OVERRIDES,
       ...clientConfigOverrides,
       ...{ httpAgent, httpsAgent },
+      proxy: false,
```

Comment:
Is proxy enabled by default in `axios.create({...})`? 
We should not change that, as it might cause irregular behaviour of our httpClient in the existing Applications.